index.html

```diff
@@ -15,38 +15,224 @@
 </head>
 
 <body>
-  <div class="wrapper">
-    <header>
-      <div class="con1">
-        <a class="logo-title" href="#">
-          <h3>My Logo</h3>
-        </a>
-        <nav>
-          <a href="#"><img class="ham-img" src="./images/ham-menu.png" alt="Menu"></a>
-        </nav>
-      </div>
+  <main class="wrapper">
+    <header class="con1">
+      <a class="logo-title" href="#">My Logo</a>
+      <nav>
+        <a href="#"><img class="ham-img" src="./images/ham-menu.png" alt="Menu"></a>
+      </nav>
     </header>
 
-
     <section class="Homepage">
       <h1 class="hero-title">I’m Steve
         Glad to see you!</h1>
       <p class="hero-description">I’m a software developer! I can help you build a product, feature or website. Look
         through some of my work and experience! If you like what you see and have a project you need coded, don’t
         hesitate to contact me.</p>
       <h3 class="connect">LET’S CONNECT</h3>
-      <div class="icon-container">
-        <ul class="icon-list">
-          <li><a class="social-icons" href="#"><img src="./images/Twitter.png" alt="Twitter"></a></li>
-          <li><a class="social-icons" href="#"><img src="./images/Linkedin-icon.png" alt="Linkedin"></a></li>
-          <li><a class="social-icons" href="#"><img src="./images/Medium.png" alt="Medium"></a></li>
+      <ul class="icon-list">
+        <li><a class="social-icons" href="#"><img src="./images/Twitter.png" alt="Twitter"></a></li>
+        <li><a class="social-icons" href="#"><img src="./images/Linkedin-icon.png" alt="Linkedin"></a></li>
+        <li><a class="social-icons" href="#"><img src="./images/Medium.png" alt="Medium"></a></li>
+        <li><a class="social-icons" href="#"><img src="./images/Github.png" alt="Github"></a></li>
+        <li><a class="social-icons" href="#"><img src="./images/Angellist.png" alt="Angellist"></a></li>
+      </ul>
+    </section>
+
+    <div class="works-section-grid">
```

Comment:
- [ ] Your project looks amazing 💯 and I admire it. However, for better code organization and HTML best practices, please consider using a `section` tag instead of `div` tag to improve user interaction and better semantic HTML 😃.